x/ccv/provider/keeper/keeper.go

```diff
@@ -1190,14 +1190,17 @@ func (k Keeper) SetOptedIn(
 	chainID string,
 	providerAddr types.ProviderConsAddress,
 	blockHeight uint64,
+	power uint64,
 ) {
 	store := ctx.KVStore(k.storeKey)
 
-	// validator is considered opted in
 	blockHeightBytes := make([]byte, 8)
 	binary.BigEndian.PutUint64(blockHeightBytes, blockHeight)
 
-	store.Set(types.OptedInKey(chainID, providerAddr), blockHeightBytes)
+	powerBytes := make([]byte, 8)
+	binary.BigEndian.PutUint64(powerBytes, power)
+
+	store.Set(types.OptedInKey(chainID, providerAddr), append(blockHeightBytes, powerBytes...))
```

Comment:
Store `OptedInValidator` as a value (define it in the proto files).

Reply:
Is the recommendation here to define it in proto to simplify our life later on when we implement queries that return opted-in validators?